routes/news.routes.js

```diff
@@ -8,13 +8,69 @@ router.get(
     try {
       let findNews = await newsModel.find()
       
-      let manyNews = await findNews.map(item => ({
+      let lastNews = await findNews.map(item => ({
         id: item._id,
         name: item.name,
         img: item.pictures[0]
+      })).slice(-3)
+
+      res.json(lastNews)
+    } catch (e) {
+      res.status(500).json({ message: "its Error, try again!" })
+    }
+  }
+)
+
+router.get(
+  '/all-news',
+  async (req, res) => {
+    try {
+      let findNews = await newsModel.find()
+      
+      let allNews = await findNews.map(item => ({
+        id: item._id,
+        name: item.name,
+        img: item.pictures[0]
+      }))
```

Comment:
why we need name and img? we will get it when we go to news details route

Reply:
No, we need it in news page for news card